### PR TITLE
Customize the infrastructure index pages

### PR DESCRIPTION
**Pull request by eshaben**
This PR adds more content and external links to the index pages in the infrastructure section 🚀

### CODE REVIEW

File: infrastructure/index.md [hunk 1]
```diff
@@ -8,8 +8,36 @@ template: index-page.html
...
+Selecting your role within the Polkadot ecosystem depends on your goals, resources, and expertise. Below are detailed considerations for each role:
+
+- **Running a node**:
+    - **Purpose** - a node provides access to network data and supports API queries. It is commonly used for:
+        - **Development and testing** - offers a local instance to simulate network conditions and test applications
+        - **Production Use** - acts as a data source for dApps, clients, and other applications needing reliable access to the blockchain
+    - **Requirements** - moderate hardware resources to handle blockchain data efficiently
+    - **Responsibilities** - a node’s responsibilities vary based on its purpose:
+        - **For Development and testing** - enables developers to test features, debug code, and simulate network interactions in a controlled environment
+        - **For Production Use** - provides consistent and reliable data access for dApps and other applications, ensuring minimal downtime
```
Comment (nhussein11) [inline infrastructure/index.md:27]: seems that "for" can be skipped here, wdyt?

```suggestion
        - **Development and testing** - enables developers to test features, debug code, and simulate network interactions in a controlled environment
        - **Production use** - provides consistent and reliable data access for dApps and other applications, ensuring minimal downtime
```

File: infrastructure/index.md [hunk 2]
```diff
@@ -8,8 +8,36 @@ template: index-page.html
...
+- Ready to get started? Jump to [In This Section](#in-this-section) to get started
+
+## Choosing the Right Role
+
+Selecting your role within the Polkadot ecosystem depends on your goals, resources, and expertise. Below are detailed considerations for each role:
+
+- **Running a node**:
+    - **Purpose** - a node provides access to network data and supports API queries. It is commonly used for:
+        - **Development and testing** - offers a local instance to simulate network conditions and test applications
+        - **Production Use** - acts as a data source for dApps, clients, and other applications needing reliable access to the blockchain
```
Comment (nhussein11) [inline infrastructure/index.md:23]:
```suggestion
        - **Production use** - acts as a data source for dApps, clients, and other applications needing reliable access to the blockchain
```

File: infrastructure/running-a-validator/operational-tasks/index.md [hunk 3]
```diff
@@ -8,8 +8,25 @@ template: index-page.html
 
 # Operational Tasks
 
-Learn how to manage your Polkadot validator node, including monitoring performance, running a backup validator for maintenance, and rotating keys.
+Running a Polkadot validator node involves several key operational tasks to ensure secure and efficient participation in the network. In this section, you'll learn how to manage and maintain your validator node by monitoring its performance, conducting regular maintenance, and ensuring high availability through strategies like running a backup validator. You'll also find instructions on rotating your session keys to enhance security and minimize vulnerabilities. Mastering these tasks is essential for maintaining a reliable and trusted presence within the Polkadot ecosystem.
```
Comment (nhussein11) [inline infrastructure/running-a-validator/operational-tasks/index.md:11]:
```suggestion
Running a Polkadot validator node involves several key operational tasks to ensure secure and efficient participation in the network. In this section, you'll learn how to manage and maintain your validator node by monitoring its performance, conducting regular maintenance, and ensuring high availability through strategies like running a backup validator. You'll also find instructions on rotating your session keys to enhance security and minimize vulnerabilities. Mastering these tasks is essential for maintaining a reliable and trusted presence within your network.
```

**Review by nhussein11 — APPROVED** [review]
LGTM! 🚀

**Review by albertov19 — APPROVED** [review]
Wow this is great!! LGTM!

**Pull request merged**